src/components/send-button/index.js

```diff
@@ -217,10 +226,12 @@ export default compose( [
 						{ adsWarning ? (
 							<Notice isDismissible={ false }>
 								{ adsWarning }{' '}
-								<a
-									href={ `/wp-admin/edit.php?post_type=${ NEWSLETTER_CPT_SLUG }&page=newspack-newsletters-ads-admin` }
+								<a // eslint-disable-line react/jsx-no-target-blank
+									href={ activeAdManageUrl }
+									rel={ adLabel === 'promotion' ? 'noreferrer' : '' }
+									target={ adLabel === 'promotion' ? '_blank' : '_self' }
 								>
-									{ __( 'Manage ads', 'newspack-newsletters' ) }
+									{ __( 'Manage ' + adLabel + 's.', 'newspack-newsletters' ) }
```

Comment:
The first argument should be a string literal, not a computed value. `sprintf` should be used:
```suggestion
									{ sprintf( __( 'Manage %ss.', 'newspack-newsletters' ), adLabel ) }
```